org.garudalinux.firedragon.yml

```diff
@@ -0,0 +1,173 @@
+app-id: org.garudalinux.firedragon
+name: FireDragon
```

Comment:
there is no such property in the `manifest`. Where in the documentation does it say it is valid? So I can get it fixed.

Reply:
Copied from one.ablaze.floorp manifest. Fixed in 178c42df127cd855ee5e3cb010cbc0d6cf5ed307.